Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml

```diff
@@ -25,6 +25,11 @@ configuration:
   name: incident_type
   required: false
   type: 13
+- defaultvalue: 3 days
```

Comment:
Why a default of 3 days? What will happen to integrations already in use?

Reply:
Integrations that are already in use shouldn't use this because their highest_id should be lt 0